### ISSUE TITLE
get-stream@7.0.0 requires Node.js >= 16

### ISSUE THREAD

**Issue by XhmikosR** [jaredwray/cacheable-request#252]
Hey, @jaredwray.

The get-stream update to v7.0.0 is a breaking change as it requires Node.js>= 16: https://github.com/sindresorhus/get-stream/releases/tag/v7.0.0

Could you downgrade it and cut a new patch release?

Thanks!

**Comment by s100** [comment]
Yes, we have found that the upgrade from `cacheable-request@10.2.10` to `cacheable-request@10.2.11` drops support for Node.js 14, which is a breaking change, which has indeed broken us, as we still use Node.js 14. Normally, under the constraints of [Semantic Versioning](https://semver.org/), a breaking change like this would warrant a major version bump, to `cacheable-request@11`. Please could you revert this upgrade?

Alternatively, if this package doesn't follow SemVer then... I guess that would be worth documenting in your README? It's something downstream consumers would need to know about before using this package.

**Comment by jaredwray** [comment]
This has been resolved with a new update `cacheable-request@10.2.12`

**Comment by XhmikosR** [comment]
Can you also please push the git tag just for completeness?
